arista-dcs-7124S/workflows/deploy-startup-config-graph.json

```diff
@@ -0,0 +1,31 @@
+{
+    "friendlyName": "Deploy Arista switch",
+    "injectableName": "Graph.Switch.Arista.Deploy",
+    "options": {
+        "deploy-config": {
+            "startupConfig": null,
+            "kickstartImage": null,
+            "bootImage": null
+        }
+    },
+    "tasks": [
+        {
+            "label": "deploy-startup-config",
+            "taskDefinition": {
+                "friendlyName": "Deploy switch config",
+                "injectableName": "Task.Inline.Switch.Deploy.Arista",
+                "implementsTask": "Task.Base.Linux.Commands",
+                "options": {
+                    "startupConfig": null,
+                    "startupConfigUri": "{{ api.base }}/templates/{{ options.startupConfig }}",
+                    "commands": [
+                        {
+                            "downloadUrl": "/api/1.1/templates/arista-deploy-startup-config.py"
```

Comment:
Actually - the downloadUrl value is the path to the deploy script (arista-deplay-startup-config.py).  The startupConfigUri is the path of the startup-config script.  So I don't think I can do:

`"downloadUrl": "{{ startupConfigUri }}"`

I think that's what you were saying.  I think I could do:

`"downloadUrl": "{{ api.base }}/templates/arista-deploy-startup-config.py"`

if that is better.